docs/src/piccolo/query_clauses/batch.rst

```diff
@@ -20,6 +20,19 @@ responses.
         async for _batch in batch:
             print(_batch)
 
+There is also an option to pass a additional database nodes (e.g. read replicas) as a 
+node argument to ``batch`` method.
```

Comment:
We should mention it's just for Postgres at the moment.

Reply:
Great, thanks. I will change that.

I think the test example I wrote in the comments is not good because I don't think it does anything. There is currently no synchronous version of batch and I don't know how to test this properly. Can you check that?

Reply:
Yes, I'll have a look.

Reply:
Didn't have time to finish looking into this - will look in the morning.